src/containers/IntentService.jsx

```diff
@@ -36,23 +37,32 @@ export default class IntentService extends Component {
           disableSuccessTimeout: !!data.disableSuccessTimeout
         })
 
-        if (!data || !data.slug) {
+        if (!data) {
           throw new Error('Unexpected data from intent')
         }
 
-        return this.store.fetchKonnectorInfos(data.slug)
+        if (data.slug) {
+          return this.store.fetchKonnectorInfos(data.slug)
```

Comment:
To always return an array of konnectors, I suggest to do here:
```js
return this.store.fetchKonnectorInfos(data.slug).then(konnector => [konnector])
```
What do you think ?